tests/utilities/dist.py

```diff
@@ -29,11 +29,10 @@ def call_training_script(module_file, cli_args, method, tmpdir, timeout=60):
 
     # need to set the PYTHONPATH in case pytorch_lightning was not installed into the environment
     env = os.environ.copy()
-    env['PYTHONPATH'] = f'{pytorch_lightning.__file__}:' + env.get('PYTHONPATH', '')
+    env['PYTHONPATH'] = env.get('PYTHONPATH', '') + f'{pytorch_lightning.__file__}:'
```

Comment:
I am not sure why the self-file shall not be the first one... ?

Reply:
In my setup, I installed pytorch-lightning and its dependencies. When test_model is launched as subprocess, it uses `tests` module from the virtualenv instead of the local 'tests' as supposed (and `PYTHONPATH` contains local `pytorch-lightning` first)